server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/SelenideConfig.java

```diff
@@ -53,6 +53,7 @@ static Browser of(String s) {
 
   public static WebDriver configure(Orchestrator orchestrator) {
     String browserKey = orchestrator.getConfiguration().getString("orchestrator.browser", Browser.FIREFOX.name());
+    System.out.println("---- Browser: " + browserKey);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106)